<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryView</class>
 <widget class="QWidget" name="EntryView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="lbl_description">
     <property name="text">
      <string>Description</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLabel" name="lbl_name">
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lbl_file">
     <property name="text">
      <string>File:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lbl_fileStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QTableView" name="tableView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="btn_exportFile">
     <property name="text">
      <string>Export</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="lbl_fileName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>(filename)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_exportFile</sender>
   <signal>clicked()</signal>
   <receiver>EntryView</receiver>
   <slot>_export_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_export_file()</slot>
 </slots>
</ui>
